app/__init__.py

```diff
@@ -292,6 +298,7 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.babel.localeselector(get_locale)
     application.jinja_env.add_extension('jinja2.ext.i18n')
+    application.jinja_env.cache = {}
```

Comment:
Because of the way the app is *not* launched in Apache, it uses the WSGI hooks via application.py (not `__main__`) so not sure exactly how it loads the app on each request (caches may get wiped/not persisted) someone who understands WSGI better than me would probably know...